nodes/project-link.js

```diff
@@ -746,11 +741,12 @@ module.exports = function (RED) {
         node.project = n.project
         node.subTopic = n.topic
         node.topic = buildLinkTopic(node, node.project, node.subTopic, false, false)
-        node.responseTopic = buildLinkTopic(node, node.project, node.subTopic, false, true)
+        node.responseTopic = buildLinkTopic(node, RED.settings.flowforge.projectID, node.subTopic, false, true)
         let timeout = parseFloat(n.timeout || '30') * 1000
         if (isNaN(timeout)) {
             timeout = 30000
         }
+        const messageEvents = {}
```

Comment:
```suggestion
        /** @type {MessageEvents} */
        const messageEvents = {}
```

Restore type hint - better dev experience with autocompletion and property listing of object and properties` properties.